hack/.deployUtils

```diff
@@ -321,13 +321,28 @@ deployPrometheusForFederation() {
 }
 
 installAPIDashboards() {
+  local PROMETHEUS_FOR_FEDERATION_API_DASHBOARDS_KUSTOMIZATION_DIR="$2"
+  local PROMETHEUS_FOR_FEDERATION_API_DASHBOARDS_GRAFANA_PATCH="$3"
```

Comment:
Im thinking a format check on the env var to ensure its in a way we expect

Reply:
I didn't add any validation as it's not done anywhere else in this file where env vars are passed in.
If the value is invalid in some way, the script will likely fail with an error.
At this time, the values being passed in to this function are not influenced by the person running the script, so they are controlled.